packages/core/crates/core-misc/src/environment.rs

```diff
@@ -33,6 +38,8 @@ impl ToString for EnvironmentType {
 /// the client is going to use
 #[derive(Deserialize, Serialize, Clone)]
 #[serde(deny_unknown_fields)]
+#[cfg(feature = "wasm")]
+#[wasm_bindgen(getter_with_clone)]
```

Comment:
The problem with this approach is that when you disable `wasm` feature, it removes the entire definition of the `NetworkOptions` class (not just of the `#[wasm_bindgen(getter_with_clone)]` annotation) and the entire code becomes unbuildable.